modules/platforms/cpp/ignite/client/detail/node_connection.h

```diff
@@ -261,6 +261,7 @@ class node_connection : public std::enable_shared_from_this<node_connection> {
 
     /** Configuration. */
     const ignite_client_configuration &m_configuration;
+    void on_observable_timestamp_changed(int64_t observable_timestamp) const;
```

Comment:
This has to be moved upper, to methods section, and given some simple description.

Reply:
Fixed.